pm/sender_test.go

```diff
@@ -78,6 +78,51 @@ func TestCreateTicket_GivenNonexistentSession_ReturnsError(t *testing.T) {
 	}
 }
 
+func TestCreateTicket_GetSenderInfoError_ReturnsError(t *testing.T) {
+	sender := defaultSender(t)
+	sm := sender.senderManager.(*stubSenderManager)
+	sm.err = errors.New("GetSenderInfo error")
+
+	sessionID := sender.StartSession(defaultTicketParams(t, RandAddress()))
+	_, _, _, err := sender.CreateTicket(sessionID)
+	assert.EqualError(t, err, sm.err.Error())
+}
+
+func TestCreateTicket_EVTooHigh_ReturnsError(t *testing.T) {
+	sender := defaultSender(t)
+	sender.maxEV = big.NewRat(100, 1)
+
+	ticketParams := TicketParams{
+		Recipient:         RandAddress(),
+		FaceValue:         big.NewInt(202),
+		WinProb:           new(big.Int).Div(maxWinProb, big.NewInt(2)),
+		Seed:              big.NewInt(3333),
+		RecipientRandHash: RandHash(),
+	}
+	sessionID := sender.StartSession(ticketParams)
+	_, _, _, err := sender.CreateTicket(sessionID)
```

Comment:
Should we check that the returned ticket is in fact `nil` or would that be assertion overkill?

Reply:
Seems ok not to check the ticket pointer right now since the expectation is that if the error is non-nil then none of the other values are safe to use for the `CreateTicket()` method. WDYT?

Reply:
Seems reasonable to me 👍